lib/ansiblelint/utils.py

```diff
@@ -272,7 +272,11 @@ def extract_from_list(blocks, candidates):
     for block in blocks:
         for candidate in candidates:
             if candidate in block:
-                results.extend(block[candidate])
+                if hasattr(block[candidate], '__iter__'):
```

Comment:
presumably `if isinstance(block[candidate], list)` would work equally well - do we expect anything other than a (possibly empty) list?